Deploy/IaCTM/data.tf

```diff
@@ -0,0 +1,7 @@
+// Current connection/authorization information for Terraform AzureRM provider
+data "azurerm_client_config" "current" {}
+
+// Retrieve Microsoft WebApps / WebSites Application Principal, because ObjectID is different in every tenant.
+data "azuread_service_principal" "MicrosoftWebSites" {
+  application_id = "abfa0a7c-a6b6-4736-8310-5855508787cd"
```

Comment:
Will this ID be replaced later by another value? I just wondered! 😏

Reply:
Thanks for checking that! While not good to have hardcoded values :) in this case it is needed and will not be replaced (at least not without a lot of effort elsewhere as well).

Please refer to the Terraform AzureRM doc:
https://www.terraform.io/docs/providers/azurerm/r/app_service_certificate.html#argument-reference

App Service will need access to KeyVault in order to use the KeyVault linking functionality.